jwst/assign_mtwcs/assign_mtwcs_step.py

```diff
@@ -22,17 +21,29 @@ class AssignMTWcsStep(Step):
     """
 
     spec = """
-    suffix             = string(default='assign_mtwcs')        # Default suffix for output files
-    output_use_model   = boolean(default=True)      # When saving use `DataModel.meta.filename`
+        suffix = string(default='assign_mtwcs')    # Default suffix for output files
+        output_use_model = boolean(default=True)   # When saving use `DataModel.meta.filename`
     """
+
     def process(self, input):
+
         if isinstance(input, str):
             input = datamodels.open(input)
+
+        # Can't apply the step if we aren't given a ModelContainer as input
         if not isinstance(input, datamodels.ModelContainer):
             log.warning("Input data type is not supported.")
             # raise ValueError("Expected input to be an association file name or a ModelContainer.")
             input.meta.cal_step.assign_mtwcs = 'SKIPPED'
             return input
 
+        # Apply the step
         result = assign_moving_target_wcs(input)
+
+        # Check for a null result
+        if result is None:
+            for model in input:
+                model.meta.cal_step.assign_mtwcs = 'SKIPPED'
+            return input
```

Comment:
This is a side comment, if you want to leave it as is that's fine. And I wrote this initially so it's really on me. 

The code would be a little bit more readable and symmetric if `assign_moving_target_wcs` returned the input model and set the status instead of returning None.